app/commands.py

```diff
@@ -914,3 +914,26 @@ def populate_annual_billing_with_defaults(year, missing_services_only):
 
     for service in active_services:
         set_default_free_allowance_for_service(service, year)
+
+
+@notify_command(name='tmp-backfill-custom-broadcast-areas')
+def tmp_backfill_custom_broadcast_areas():
+    from app import db
```

Comment:
`db` is already being imported at the top of the file, and it would be nicer to have `BroadcastMessage` imported at the top if possible too (where we import other models). However, if the plan is to remove this code after the command is run, then I don't think it matters too much if it's not changed (I can see how importing here could make the change easier to undo)

Reply:
Oh yes, I could remove the `db` one, but yes the idea is to make it standalone and easy to remove.